src/components/application_manager/src/helpers/application_helper.cc

```diff
@@ -48,9 +48,9 @@ void DeleteSubmenus(ApplicationSharedPtr app, ApplicationManager& app_manager) {
 void DeleteChoiceSets(ApplicationSharedPtr app,
                       ApplicationManager& app_manager) {
   auto accessor = app->choice_set_map();
-  const auto& choices = accessor.GetData();
+  const auto choices = accessor.GetData();
 
-  for (const auto& choice : choices) {
+  for (const auto choice : choices) {
```

Comment:
```suggestion
  for (const auto& choice : choices) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e